### DISCUSSION TITLE
Freeplane - Created mm-file does not open

### DISCUSSION THREAD

**Discussion by Phil8008** [freeplane/freeplane#330] [Help]
Hi, new to this forum and honestly no idea if my question is placed here correctly, however, this site was indicated to provide help for Freeplane problems. So here it is:

I have created my first mindmap with Freeplane in .mm-format.
Was quite a bit of work. Upon attempt to continue the work I could not open the file anymore and the software tells me, that this mindmap was created with an unknown program.

![image](https://user-images.githubusercontent.com/104426716/165270469-aa0debc0-db9d-453c-a107-2cf707879013.png)

I already reinstalled the program, but same error message.
Can anyone help? I do not want to create everything again and run the risk, that it vanishes again...

Thanks - Phil

**Comment by macmarrum** [comment]
Hi  @Phil8008 and welcome to the forum!

I know the feeling of loosing one's content and I want to help you. I would need to have the actual .mm file that causes the error. 
If you'd like to avoid attaching it here (zipped), please feel free to contact me on Discord → `macmarrum#3214`

**Reply by Phil8008** [reply]
Hi and thanks for the quick response. 
Can't attach the file here - system says "We don't support that file type".
I have no Discord account - if possible I do not want to open one only for this one topic. Any other ideas?

Thanks - Philipp

**Reply by macmarrum** [reply]
Yep, Github Discussions is limited in this way – https://github.com/freeplane/freeplane/discussions/255

You have 2 options here:
1. Zip the .mm file and attach the .zip file here (using your favorite utility, e.g. 7-zip)
2. Rename the .mm file, changing its extension to .txt and attach that one.

**Reply by Phil8008** [reply]
There we go:
[CTC Mainz MindMap 2022-03-11.zip](https://github.com/freeplane/freeplane/files/8562106/CTC.Mainz.MindMap.2022-03-11.zip)

**Reply by macmarrum** [reply]
The attached file has no meaningful content (only unrecognizable garbage).

Let's see if there are any backup files available (Freeplane creates backup copies at regular intervals).

Could you please have a look in `<user-dir>/.backup/` – you can open `<user-dir>` from Freeplane `Tools->Open user directory`. There are probably several backups done. Copy the latest file to another directory, rename it (to end in .mm) and open it to see if it has your content.

**Reply by Phil8008** [reply]
There is nothing in there - probably the folder was overwritten when redoing the installation.
As I used the uninstall there is also nothing in the bin to recover - guess that's a goner then...

**Reply by nnako** [reply]
Hi @Phil8008 , sorry for your loss.

Maybe a last thing could be...

shortly,

- search you computer's drives for any MM file and check if an old / recent version of your map still hangs around, somewhere.

for the future,

- move the Freeplane's standard ".backup" location (already mentioned by @macmarrum) away from your installation folder tree as soon as you installed Freeplane. when keeping it within your installed folders, you risk loosing them when re-installing the same version.
- maybe you could consider to use the "portable" versions of Freeplane where you can individually select the installation folder location and thus e.g. simultaneously work with multiple Freeplane versions on the same computer.
- always keep your data (in this case your Freeplane mindmaps) within a separate location outside the installation tree of Freeplane (and any other application). on Windows systems, preferrably keep them on a data partition or within your secured documents folder (somewhere below `c:\Users\<user>\...`)
- when you work with removable (and often removed) memory sticks e.g. for manual synchronizing of files between computers, please keep in mind that this tends to corrupt data, especially when working with auto-synchronization through NextCloud or other collaboration systems. I had loads of issues, here.
- always save your files, frequently, using Freeplane's "save" menu entry or corresponding buttons / keys. and backup your files when you have invested your precious time into creating them, on a regular basis (outside your installation trees).

And, yes, please report any oddity within this forum. :-)

**Comment by Phil8008** [comment]
Hi, thanks a lot for your suggestions.

 

Unfortunately no files found on the entire harddisk.

The rest I will consider.

 

 

Von: nnako ***@***.***> 
Gesendet: Sonntag, 1. Mai 2022 10:56
An: freeplane/freeplane ***@***.***>
Cc: Phil8008 ***@***.***>; Mention ***@***.***>
Betreff: Re: [freeplane/freeplane] Freeplane - Created mm-file does not open (Discussion #330)

 

Hi @Phil8008 <https://github.com/Phil8008>  , sorry for your loss.

Maybe a last thing could be...

shortly,

*	search you computer's drives for any MM file and check if an old / recent version of your map still hangs around, somewhere.

for the future,

*	move the Freeplane's standard ".backup" location (already mentioned by @macmarrum <https://github.com/macmarrum> ) away from your installation folder tree as soon as you installed Freeplane. when keeping it within your installed folders, you risk loosing them when re-installing the same version.
*	maybe you could consider to use the "portable" versions of Freeplane where you can individually select the installation folder location and thus e.g. simultaneously work with multiple Freeplane versions on the same computer.
*	always keep your data (in this case your Freeplane mindmaps) within a separate location outside the installation tree of Freeplane (and any other application). on Windows systems, preferrably keep them on a data partition or within your secured documents folder (somewhere below c:\Users\<user>\...)
*	when you work with removable (and often removed) memory sticks e.g. for manual synchronizing of files between computers, please keep in mind that this tends to corrupt data, especially when working with auto-synchronization through NextCloud or other collaboration systems. I had loads of issues, here.
*	always save your files, frequently, using Freeplane's "save" menu entry or corresponding buttons / keys. and backup your files when you have invested your precious time into creating them, on a regular basis (outside your installation trees).

And, yes, please report any oddity within this forum. :-)
